kubejs/server_scripts/enigmatica/kubejs/base/unification/unify_dyes.js

```diff
@@ -1,22 +1,24 @@
 //priority: 900
 onEvent('recipes', (event) => {
+    const id_prefix = 'enigmatica:base/unification/unify_dyes/';
+
     dyeSources.forEach((recipe) => {
-        botania_dye_pestle_mortar(event, recipe);
-        create_dye_milling(event, recipe);
-        immersiveengineering_dye_crusher(event, recipe);
-        //integrateddynamics_dye_squeezing(event, recipe);
-        mekanism_dye_enriching(event, recipe);
-        mekanism_pigment_extracting(event, recipe);
-        pedestals_dye_crushing(event, recipe);
-        thermal_dye_centrifuge(event, recipe);
-        atum_quern_milling(event, recipe);
-        shapeless_dye_crafting(event, recipe);
-        occultism_dye_crushing(event, recipe);
+        botania_dye_pestle_mortar(event, recipe, id_prefix);
+        create_dye_milling(event, recipe, id_prefix);
+        immersiveengineering_dye_crusher(event, recipe, id_prefix);
+        //integrateddynamics_dye_squeezing(event, recipe, id_prefix);
```

Comment:
please also remove this since you removed the function

Reply:
I only commented out that function to signify it wasn't in use, you'll have to ask whomever originally commented out line 9 on what to do with it

Reply:
Odd, I tried to delete this comment ^^' I noticed you hadn't removed anything later. My bad!

Reply:
there were 3 instances of this comment :D